### DISCUSSION TITLE
how is the scrollbar styled?

### DISCUSSION THREAD

**Discussion by Fuseteam** [twbs/bootstrap#31542] [General] [answered]
i tried reproducing the table styling in bootstrap 4 in codepen in order to share and gather opinions for a solution i found for a problem i encountered with overflow and position sticky.

i have almost reproduced it in codepen with the exception of the scrollbars

[codepen](https://codepen.io/fuseteam/pen/BaKRdZY?editors=1100)
![2020-08-27-101247_565x830_scrot](https://user-images.githubusercontent.com/10421851/91448777-b16d9d00-e850-11ea-8d34-408145c830e7.png)
bootstrap 4
![2020-08-27-102422_377x463_scrot](https://user-images.githubusercontent.com/10421851/91448834-c34f4000-e850-11ea-8409-ed7d6869b0d7.png)

**Comment by mdo** [comment, marked as answer]
Scrollbars are coming from browser styling—we don't touch those in Bootstrap itself. It is possible to customize them though.

**Reply by Fuseteam** [reply]
hmm then i currently have no idea where the scrollbar styling in my app is coming from both of the above screenshots are made in firefox a docker image for second can found on my [lemp-webapp](https://github.com/fuseteam/lemp-webapp) repository

**Reply by Fuseteam** [reply]
ah now i see thanks for the hint, appearently the difference is firefox's responsive mode. do you any tips on styling it, if i may ask?

**Reply by mdo** [reply]
Check out CSS Tricks: https://css-tricks.com/the-current-state-of-styling-scrollbars/

**Reply by Fuseteam** [reply]
thank yoo
